core/templates/dev/head/services/site-analytics.service.ts

```diff
@@ -17,199 +17,224 @@
  * the learner and editor views.
  */
 
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+
+const constants = require('constants.ts');
+import { WindowRef } from 'services/contextual/window-ref.service';
+
+declare global {
```

Comment:
Move this to `typings/custom-window-defs.d.ts`.

Reply:
Done